scripts/ingests/sanghi23/Sanghi23-Ingest.py

```diff
@@ -0,0 +1,118 @@
+#SIMPLE & Astrodb Packages
+# from astrodb_utils.publications import ingest_publication
+from astrodb_utils import load_astrodb, logger
+from astrodb_utils.publications import ingest_publication
+from astrodb_utils.sources import ingest_source, find_source_in_db, ingest_names
+# from simple.utils.spectra import * 
+from simple.schema import *
+from simple.schema import REFERENCE_TABLES
+
+#Additional Packages
+import pandas as pd
+
+# logger.setLevel("DEBUG")
+
+db = load_astrodb("SIMPLE.sqlite", recreatedb=True,  reference_tables=REFERENCE_TABLES)
+
+# Ingest New Sources 
+# ingest_publication(
+#     db,
+#     doi = "10.3847/1538-4357/ad0b12")
+
+newsources = pd.read_csv('missing_sources.csv') # Read in new sources
+for _, source in newsources.iterrows():
+    try :
+        ingest_source(
+            db,
+            source = source['name'],
+            ra = source['ra_j2000_formula'],
+            dec = source['dec_j2000_formula'],
+            reference = source['reference'],
+        )
+        print(f"Source {source['name']} ingested")
+    except: 
+        print(f"Source {source['name']} already in database")
+        continue
+
+## Ingest New Fundamental Parameters
+#evo_cols = ["name", "name_simbadable", "ra_j2000_formula", "dec_j2000_formula", "teff_evo", "teff_evo_err", "radius_evo", "radius_evo_err", "mass_evo", "mass_evo_err", "logg_evo", "logg_evo_err"]
+evo_table = pd.read_csv('FundamentalPropertiesbyEvoModel.csv')
+
+modeled_parameters_evo_ingest_dict = [
+    {'sourceName': row['name'],
+     'sourceAltName': row['name_simbadable'],
+     'ra': row['ra_j2000_formula'], 
+     'dec': row['dec_j2000_formula'],
+     'Radius': {'value': row['radius_evo'], 'value_error': row['radius_evo_err'], 'parameter': "radius", 'unit': 'R_jup', "model": "evolutionary"},
+     'log_g': {'value': row['logg_evo'], 'value_error': row['logg_evo_err'], 'parameter': "log g", 'unit': 'dex', "model": "evolutionary"},
+     'T_eff': {'value': row['teff_evo'], 'value_error': row['teff_evo_err'], 'parameter': "T eff", 'unit': 'K', "model": "evolutionary"},
+     'Mass': {'value': row['mass_evo'], 'value_error': row['mass_evo_err'], 'parameter': "mass", 'unit': 'M_jup', "model": "evolutionary"},
+     'reference': "Sang23"
+     } for _, row in evo_table.iterrows()
+]
```

Comment:
This is very hard to read.  Try formatting with Black.  https://code.visualstudio.com/docs/python/formatting